AUTHORS.md

```diff
@@ -0,0 +1,23 @@
+# Authors
+
+The Tock Project Developers include dozens of contributors, some on behalf of
+their university, company, or organization. The most up to date list of contributors is available in the git
+history: `git shortlog -s -n`.
+
+Organizations that have contributed code to Tock include:
+
+  * Atihita
+  * Chalmers University
+  * Google
+  * Helium
+  * MIT
+  * MIT Lincoln Laboratory
+  * Princeton University
+  * Stanford University
+  * University of California, Berkeley
+  * University of California, San Diego
+  * University of Michigan
+  * University of Virginia
```

Comment:
Can we add WD? WD is a top 10 contributor now :)

Reply:
Yes!! Done :)